diesel/src/pg/expression/array.rs

```diff
@@ -0,0 +1,64 @@
+use std::marker::PhantomData;
+use expression::{AppearsOnTable, Expression, IntoSingleTypeExpressionList, SelectableExpression};
+
+#[derive(Debug)]
+pub struct Array<T, ST> {
+    elements: T,
+    _marker: PhantomData<ST>,
+}
+
+pub fn array<ST, T>(elements: T) -> Array<T::Expression, ST>
```

Comment:
It would be great to add some small documentation with an example to this function.

Reply:
Good point, thank you! Added in bb35442.